liquidctl/driver/commander_core.py

```diff
@@ -99,7 +114,27 @@ def set_speed_profile(self, channel, profile, **kwargs):
         raise NotSupportedByDriver
 
     def set_fixed_speed(self, channel, duty, **kwargs):
```

Comment:
Any thoughts on displaying the fan/pump mode and target fixed speed (if set) that are set here in the status message? Maybe potentially under `--verbose`? If so is there a device that does this already or format ideas?
This can be another PR, I just had the idea.

Reply:
That would be nice.

For the pump/fan mode, take a look at the Commander Pro and Smart Device (V1): those specific devices only report PWM or DC, but you could extend that idea for whether their running in fixed PWM or on-board profile modes.  IIRC, they use enums with a suitable `__str__()` implementation for the pretty/human output.

For the target duty cycle (if set), take a look at the Corsair Platinum coolers.